src/mongocxx/options/server_api.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <mongocxx/config/prelude.hpp>
+
+#include <string>
+
+#include <bsoncxx/stdx/optional.hpp>
+#include <mongocxx/stdx.hpp>
+
+namespace mongocxx {
+MONGOCXX_INLINE_NAMESPACE_BEGIN
+
+class client;
+class pool;
+
+namespace options {
+
+///
+/// Class representing options for server API.
+///
+class MONGOCXX_API server_api {
+   public:
+    ///
+    /// Enum representing the possible values for server API version.
+    ///
+    enum class version { version_1 };
```

Comment:
Used the name `version`, as `server_api_version` would mean this is called `options::server_api::server_api_version` which seems redundant.